### ISSUE TITLE
Failed to send https request on OTP 22.1

### ISSUE THREAD

**Issue by kaaboaye** [edgurgel/httpoison#393]
## What is wrong?
When you try to make request on OTP 22.1 you will get the following error:
```ex
{:error, {:option, :server_only, :honor_cipher_order}}
```

## Fix?
Put this in your `mix.exs`
```ex
{:hackney, github: "benoitc/hackney", override: true}
```

## Reference
https://github.com/benoitc/hackney/issues/591

**Comment by codehugger** [comment]
I can verify that the hackney line proposed as a fix, works like a charm.

I was having problems with an example program from the Programming Elixir book which queries Github's api for issues.

```elixir
HTTPoison.get "https://api.github.com/repos/elixir-lang/elixir/issues"
```

## Before

```elixir
{:error, {:option, :server_only, :honor_cipher_order}}
```

## After Hackney fix

```elixir
 {..., status_code: 200}
```

**Comment by perrycate** [comment]
[As of a few days ago](https://github.com/benoitc/hackney/issues/591) Hackney cut a new version with the fix. If you'd prefer not to track the github branch directly, you can now use this in your mix.exs instead:

`{ :hackney, ">= 1.15.2", override: true}`

**Comment by glenwinters** [comment]
This is fixed by the hackney upgrade in #394.

**Comment by edgurgel** [comment]
A new HTTPoison version was released as well so if you use HTTPoison 1.6+ it will pickup hackney 1.15.2 onwards.

So either `mix deps.update hackney` or `mix deps.update httpoison`
